src/config/dynamic-theme-fixes.config

```diff
@@ -3783,6 +3783,10 @@ CSS
 body {
      background-image: none !important;
 }
+body {
+    background: none !important;
```

Comment:
We already have a background-image: none, I'm not sure if this has nay more effect.

Reply:
It does -
Essentially, `background` is a shorthand for any `background-` property.

On at least some pages, dndbeyond only sets `background` - It appears to be set mainly across multiple `@media` viewport width queries.

I believe the reason that `background-image` on its own isn't working is that, at least for some pages - the light background is actually set by the `background-colour` component of the `background` statement

 `background:  var(--theme-page-bg-image-1024) no-repeat center 114px,
var(--theme-page-bg-image-base) var(--theme-page-bg-color,
#f9f9f9);` 

If I comment out the `body` selector like so:

```css
================================

dndbeyond.com

CSS
.mon-stat-block,
.mon-stat-block::before,
.mon-stat-block::after,
body {
     background-image: none !important;
}
/*
body {
    background: none !important;
}
*/
.read-aloud-text,
.more-info::after,
.details-container::after {
     border-image: none !important;
}
::selection {
    background-color: var(--darkreader-selection-background) !important;
}

================================
```

In dark mode, the page appears like this:

![image](https://user-images.githubusercontent.com/39424834/133958568-e3a74ae0-ec1a-427f-b5ff-a1d5b0660534.png)

From devtools, if I right click on the `<body ...>` element, from the `Elements` tab, and click "Copy" > "Copy Styles", I get the following styles for the element.

```css
-webkit-text-size-adjust: 100%;
--darkreader-neutral-background: #131516;
--darkreader-neutral-text: #d8d4cf;
--darkreader-selection-background: #004daa;
--darkreader-selection-text: #e8e6e3;
--theme-page-bg-image-base: url(https://media-waterdeep.cursecdn.com/attachments/0/84/background_texture.png);
--theme-page-bg-image-1024: url(https://media-waterdeep.cursecdn.com/attachments/2/195/lmop1k.jpg);
--theme-page-bg-image-1921: url(https://media-waterdeep.cursecdn.com/attachments/2/196/lmop2k.jpg);
--theme-page-bg-image-2561: url(https://media-waterdeep.cursecdn.com/attachments/2/197/lmop4k.jpg);
--darkreader-bgimg--theme-page-bg-image-base: url("https://www.dndbeyond.com/sources/lmop/https/:////media-waterdeep/.cursecdn/.com//attachments//0//84//background_texture/.png");
--darkreader-bgimg--theme-page-bg-image-1024: url(https\:\/\/media-waterdeep\.cursecdn\.com\/attachments\/2\/195\/lmop1k\.jpg);
--darkreader-bgimg--theme-page-bg-image-1921: url(https\:\/\/media-waterdeep\.cursecdn\.com\/attachments\/2\/196\/lmop2k\.jpg);
--darkreader-bgimg--theme-page-bg-image-2561: url(https\:\/\/media-waterdeep\.cursecdn\.com\/attachments\/2\/197\/lmop4k\.jpg);
box-sizing: inherit;
outline: 0;
-webkit-tap-highlight-color: transparent;
outline-color: initial;
border-color: #736b5e;
margin: 0;
line-height: 1.4;
font-size: 13px;
font-family: Roboto,
Helvetica,
sans-serif;
overflow-y: scroll;
color: rgb(208,
204,
198);
background-image: none !important;
background: var(--theme-page-bg-image-1024) no-repeat center 114px,
var(--theme-page-bg-image-base) var(--theme-page-bg-color,
#f9f9f9);
height: 100%;
```

In the "Styles" tab of devtools, we see the following:

![image](https://user-images.githubusercontent.com/39424834/133958695-76004f7d-556b-45d2-b497-5797b649f651.png)

Reply:
I have determined a more appropiate css selector, though - I will push that

Reply:
in https://github.com/darkreader/darkreader/pull/6785/commits/c536635490dc683c75dcf6aa501e909a052ff492 

I set `background-colour` correctly, in the same block that disables `background-image`, and I remove the extra `body` selector block.

This achieves the same affect on https://www.dndbeyond.com/sources/lmop/goblin-arrows , and I also checked how it affects the `.mon-stat-block` class:

<details> <summary>Also showing .mon-stat-block</summary>

https://www.dndbeyond.com/monsters/goblin

![image](https://user-images.githubusercontent.com/39424834/133959417-af425763-6fb9-477d-9b2c-579ffca64299.png)

![image](https://user-images.githubusercontent.com/39424834/133959465-ee6ba6a8-866a-4228-9ada-a78579740128.png)

</details>

`.mon-stat-block` appears to render correctly, with the css

Reply:
Should be resolved by https://github.com/darkreader/darkreader/pull/6785/commits/c536635490dc683c75dcf6aa501e909a052ff492